test/integration/connect/envoy/case-centralconf/s1.hcl

```diff
@@ -1,16 +1,19 @@
 services {
   name = "s1"
   port = 8080
+
```

Comment:
I have `vim` set to run `hclfmt` on save for HCL files, which is what happened to some of these files. I can revert if it's undesirable.